src/ol_infrastructure/components/services/k8s.py

```diff
@@ -59,12 +59,14 @@ class OLApplicationK8sConfiguration(BaseModel):
     model_config = ConfigDict(arbitrary_types_allowed=True)
 
     project_root: Path
-    application_replicas: int = 1
+    application_replicas: int = 0
```

Comment:
I think we can probably get rid of this in favor of the `min_replicas` field, or vice versa. In the case where there are no scaling metrics passed, then we use the static replica count, otherwise we provision the HPA configs.

Reply:
OK. I can't figure out how to implement the "either replicas OR ASG" check you mentioned previously given this, so I removed it.